frontend/src/modules/dashboard/containers/ValueInfo.tsx

```diff
@@ -122,12 +130,37 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+/**
+ * @typedef ValueInfoProps
+ *
+ * Props interface for the showing value information.
+ *
+ * @prop {Props} mainContainer Main Container placed on Left side in layout with more space
+ * @prop {Props} subContainer1 Sub Container placed on Right side, with half the Main container height
+ * @prop {Props} subContainer2 Sub Container placed on Right side, with other half
+ *
+ */
 export interface ValueInfoProps {
   mainContainer: Props;
   subContainer1?: Props;
   subContainer2?: Props;
 }
 
+/**
+ * @typedef Props
+ *
+ * Props interface for the showing information.
+ *
+ * @prop {SelectorType} selector Redux Selector
+ * @prop {string} label Value label
+ * @prop {string} stateKey Unique identifier
+ * @prop {string} units Unit measurement to display
+ * @prop {boolean} isLive Config to show isLive in UI
+ * @prop {boolean} isMain Config to know if its main or sub container
```

Comment:
Reference `PressureControlMainView.tsx`, In dashboard page, On left end side, we had dynamic layout to show blocks of `ValueInfo`. Each block had configuration to show 1 vertical container (main) with 2 horizontal container(sub) placed in parallel. And also there were layout with no Sub container as well. To manage this configuration, is Main key is used. If `isMain` key is true, It would take up full space of the block and will show only 1 main container